### DISCUSSION TITLE
Suggestions for Galaxy Brain Achievements

### DISCUSSION THREAD

**Discussion by mofmofmofneko** [community/community#41645] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

To unlock this achievement, it is necessary to answer appropriately in the discussion, but there is a problem.
That is, the creator of the discussion is not included among those who can obtain achievements.

“Discussion” includes not only “questions” but also “suggestions”.

In the latter case, the proposer should be evaluated, which I think is the creator of the discussion.
So I think that the creator of the discussion should also be the target of the achievement.

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
